commands/user.go

```diff
@@ -104,30 +114,47 @@ func init() {
 	RootCmd.AddCommand(UserCmd)
 }
 
-func deactivateUsers(c client.Client, userArgs []string) {
+func userActivateCmdF(c client.Client, command *cobra.Command, args []string) error {
+	if len(args) < 1 {
+		return errors.New("Expected at least one argument. See help text for details.")
+	}
+
+	changeUsersActiveStatus(c, args, true)
+
+	return nil
+}
+
+func changeUsersActiveStatus(c client.Client, userArgs []string, active bool) {
 	users := getUsersFromUserArgs(c, userArgs)
 	for i, user := range users {
-		if user == nil {
-			printer.PrintError("Unable to find user '" + userArgs[i] + "'")
-			continue
-		}
+		err := changeUserActiveStatus(c, user, userArgs[i], active)
 
-		if user.IsSSOUser() {
-			printer.Print("You must also deactivate user " + userArgs[i] + " in the SSO provider or they will be reactivated on next login or sync.")
+		if err != nil {
+			printer.PrintError(err.Error())
 		}
+	}
+}
 
-		if _, response := c.DeleteUser(user.Id); response.Error != nil {
-			printer.PrintError("Unable to deactivate user " + userArgs[i] + ". Error: " + response.Error.Error())
-		}
+func changeUserActiveStatus(c client.Client, user *model.User, userArg string, activate bool) error {
+	if user == nil {
+		return fmt.Errorf("Can't find user '%v'", userArg)
+	}
+	if !activate && user.IsSSOUser() {
+		printer.Print("You must also deactivate user " + userArg + " in the SSO provider or they will be reactivated on next login or sync.")
 	}
+	if _, response := c.UpdateUserActive(user.Id, activate); response.Error != nil {
+		return fmt.Errorf("Unable to change activation status of user: %v", userArg)
+	}
+
+	return nil
 }
 
 func userDeactivateCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 	if len(args) < 1 {
```

Comment:
It doesn't belong specifically to this ticket, but it would be great if you could apply the `Args: cobra.MinimumNArgs(1)` to this command as well